tests/bwc/test_rolling_upgrade.py

```diff
@@ -225,6 +255,29 @@ def _test_rolling_upgrade(self, path, nodes):
                 # Add the shards of the new partition primaries
                 expected_active_shards += shards
 
+                if int(path.from_version.split('.')[0]) >= 5:
+                    c.execute(f"insert into doc.t2(a, b) values ({idx}, {idx})")
+                    c.execute("refresh table t2")
+                    c.execute(f"select a, b, c>=1 and c<=2, d>a+b from doc.t2 where a = {idx}")
```

Comment:
Use parameter placeholders, never inline values via format strings. Not that it matters here, but it is a bad habit that can lead to SQL injections.